spec/calculations/users_by_age_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe UsersByAge do
+  describe '.result_for' do
+    it 'returns sorted array of age ranges with users count withing organization' do
```

Comment:
`withing` -> `within`
rephrase to: `returns sorted age ranges with users count for organization`